src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodsServiceSpec.scala

```diff
@@ -52,7 +50,7 @@ class MethodsServiceSpec extends FreeSpec with ScalaFutures with ScalatestRouteT
       "MethodNotAllowed error is returned" in {
         Put("/methods") ~> sealRoute(routes) ~> check {
           status should equal(MethodNotAllowed)
-          responseAs[String] === "HTTP method not allowed, supported methods: GET"
+          errorReportCheck("Agora", MethodNotAllowed)
```

Comment:
Most of these tests were written when we didn't have a concept of passthrough. In my opinion, the vast majority of these tests should go away. Anything that is a passthrough - we really should only be testing that orchestration can perform the passthrough. We should not care about what is passed or what is returned - that should be something we do in when integration testing.